ceph_deploy/mon.py

```diff
@@ -536,6 +530,25 @@ def make(parser):
 #
 
 
+def get_mon_initial_members(args, error_on_empty=False, _cfg=None):
+    """
+    Read the ceph config file and return the value of mon_initial_members
+
+    Optionally, a NeedHostError can be raised if the value is None.
+    """
+    if _cfg:
+        cfg = _cfg
+    else:
+        cfg = conf.ceph.load(args)
```

Comment:
Just a nit - this can be done with
`cfg = _cfg or conf.ceph.load(args)`

You don't need to change it thought.